### ISSUE TITLE
ChatGPT_Client constructor missed ChromeOptions and Chrome

### ISSUE THREAD

**Issue by grayskripko** [ugorsahin/ChatGPT_Automation#11]
There is no strict way to pass your own settings if you have another path to chromedriver or google-chrome-stable

**Comment by ugorsahin** [comment]
It is a good point that we need a way to customize chrome initialization. Feel free to suggest a solution, I will also look for a way to enable this.

**Comment by grayskripko** [comment]
like
```
class ChatGPT_Client:
...
def __init__(
        self,
        username :str,
        password :str,
        headless :bool = True,
        cold_start :bool = False,
        verbose :bool = False,
        browser .. = None
    ):
..
if browser is None:
    self.browser = uc.Chrome(
            # driver_executable_path='/usr/bin/google-chrome',
            options=options,
            headless=headless,
            version_main=112
        )
...
```

**Comment by ugorsahin** [comment]
I addressed your problem with commit 6cde457cf3dce8b3e654e176240667dbb506a90f. Now you can pass driver executable and options to constructor, I hope that will be helpful for you.
